docs/user_manual/source/control_status_registers.rst

```diff
@@ -1556,20 +1552,21 @@ Reset Value: Not applicable
   +-------+-------------+----------------------------------------------------------------+
   | 27    | WARL (0x1)  | **DMODE**. Only debug mode can write ``tdata`` registers.      |
   +-------+-------------+----------------------------------------------------------------+
-  | 26:25 | WARL (0x0)  | Hardwired to 0.                                                |
+  | 26    | WARL (0x0)  | **UNCERTAIN**. Hardwired to 0.                                 |
+  +-------+-------------+----------------------------------------------------------------+
+  | 25    |             | **HIT1**. Forms 2-bit WARL (0x0, 0x1) bitfield  with **HIT0**. |
   +-------+-------------+----------------------------------------------------------------+
   | 24    | WARL (0x0)  | **VS**. Hardwired to 0.                                        |
   +-------+-------------+----------------------------------------------------------------+
   | 23    | WARL (0x0)  | **VU**. Hardwired to 0.                                        |
   +-------+-------------+----------------------------------------------------------------+
-  | 22    | WARL (0x0)  | **HIT**. Hardwired to 0.                                       |
+  | 22    |             | **HIT0**. Forms 2-bit WARL (0x0, 0x1) bitfield  with **HIT1**. |
```

Comment:
I am not sure if it makes sense to have hit0 and hit1 writeable, as they are merely a report from the core about what timing a certain trigger has. I see that the debug spec says the bits are WARL, but what would/should happen when SW or the debugger writes these bits?